requirements-rtd.txt

```diff
@@ -0,0 +1,88 @@
+# Requirements for ReadTheDocs
+# RTD doesn't like postgres, memcache, and doesn't need others
+
+# Django
+# 1.8 is waiting on django-simple-history
+Django==1.7.8  # rq.filter: >=1.7, <1.8
+
+# Better templates
+MarkupSafe==0.23
+Jinja2==2.7.3
+jingo==0.7.1
+
+# Django REST Framework.
+djangorestframework==3.1.3
+Markdown==2.6.2
+django-filter==0.10.0
+
+# Django extensions
+six==1.9.0
+django-extensions==1.5.5
+
+# JSON API interfaces for Django REST Framework
+git+git://github.com/jwhitlock/drf-json-api@v0.1d#egg=drf-json-api
+
+# History of changes to models
+django-simple-history==1.6.1
+
+# Configure database from env
+dj-database-url==0.3.0
+
+# Better test runner (included in settings)
+nose==1.3.7
+django-nose==1.4
+
+# Test Mocking, included in Python 3.3
+mock==1.0.1
+
+# WSGI runner, used in Heroku
+# gunicorn==19.3.0
+
+# Connect to PostgreSQL
+# psycopg2==2.6.1
+
+# Serve static files
+# static3==0.6.1
+# dj-static==0.0.6
+
+# Timezone info
+pytz==2015.4
+
+# Fast Memcache connections, and tell Heroku to install C dependencies
+# pylibmc==1.4.3
+# django-pylibmc==0.6.0
+# django-pylibmc-sasl==0.2.4
+
+# Celery - async task management
+billiard==3.3.0.20
+amqp==1.4.6
+anyjson==0.3.3
+kombu==3.0.26
+celery==3.1.18
+
+# Modified Preorder Tree Traversal
+django-mptt==0.7.4
+
+# Sorted ManyToManyField
+django-sortedm2m==0.10.0
+
+# Cached instances for Django REST Framework
+drf-cached-instances==0.3.1
+
+# CORS headers in middleware
+# django-cors-headers==1.1.0
+# Bug #50 (fixed by PR #51) prevents Firefox CORS
+git+git://github.com/ottoyiu/django-cors-headers@1e7bf86#egg=django-cors-headers
+
+# Parsing Expression Grammar, for MDN scraping
+# parsimonious==0.6.2
+# TravisCI is now using ascii as system encoding, breaking setup.py
+git+git://github.com/erikrose/parsimonious@20863d86a#egg=parsimonious
+
+# Social loging support (Firefox Accounts)
+# python-openid >= 2.2.5  # Python 2.x - unused by app
+# python3-openid >= 3.0.1 # Python 3.x - unused by app
+requests==2.7.0
+oauthlib==0.7.2
+requests-oauthlib==0.5.0
+django-allauth==0.20.0
```

Comment:
Whoa, why all this? Because we use the app itself to generate the example API responses in the docs?